ibm/mas_devops/roles/uds/tasks/install/determine-storage-classes.yml

```diff
@@ -26,6 +26,11 @@
   set_fact:
     uds_storage_class: "{{ lookup_storageclasses | ibm.mas_devops.defaultStorageClass(default_storage_classes_rwo) }}"
 
+- name: Default annotated storage if not found in default list
+  when: uds_storage_class is not defined or uds_storage_class == ""
+  set_fact:
+    uds_storage_class: "{{ lookup_storageclasses | ibm.mas_devops.getDefaultStorageClass() }}"
```

Comment:
Does UDS work with any access mode?

Reply:
UDS requires rwo storageclass. Just added this condition to check and use the storgaclass with default annotation if default_storage_classes_rwo list returns no result.

Reply:
I guess the problem is what happens if the default storage class is rwx

Reply:
I think, If the default storage class is only rwx then, pvc creation might fail.